Homework2/StackCalculator/StackCalculator/StackOnArray.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Stack;
+
+public class StackOnArray : IStack
+{
+    public StackOnArray()
+    {
+        this.array = new double[10];
+    }
+
+    private double[] array;
+    private int position = 0;
+
+    public bool IsEmpty()
+    {
+        return position == 0;
+    }
```

Comment:
```suggestion
    public bool IsEmpty()
        => position == 0;
```
И я бы это свойством сделал, оно же не делает содержательных вычислений и просто возвращает состояние объекта. Хотя я думаю, что во времена, когда Вы это сдавали, мы свойства не проходили ещё :)